core/templates/components/forms/schema-based-editors/schema-based-custom-editor.component.spec.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for schema-based editor component for custom values
+ */
+
+import { NO_ERRORS_SCHEMA } from '@angular/core';
```

Comment:
Done, unused in two files of that directory, hence removed but in rest it used for `oppiaFocusOn` , `objType`  property etc.

Reply:
Sorry, I don't think this answers my question. Why are we using it, and can we not use it?

Reply:
`NO_ERRORS_SCHEMA` is used to avoid dependencies of unit tests on child components, for this case that is `object-editor`. Using `NO_ERRORS_SCHEMA` karma will consider `object-editor` as a standard HTML tag. So, there is no need to load the actual component which avoids dependency